File: opbp-backend/src/main/scala/br/com/eteg/opbp/DatabaseLoader .scala
package br.com.eteg.opbp

import org.springframework.stereotype.Component

import br.com.eteg.opbp.repository.TodoRepository
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.CommandLineRunner
import br.com.eteg.opbp.entities.Todo

@Component
class DatabaseLoader @Autowired() (private val todoRepository: TodoRepository) extends CommandLineRunner {
  
  def run(args: String*) = {
    for (i <- 1 to 10) {
      val todo = new Todo()

      todo.text = s"Tarefa #$i"
      todo.completed = false

      todoRepository.save(todo)
    }
  }
}

File: opbp-backend/src/main/java/br/com/eteg/opbp/DatabaseLoader.java
package br.com.eteg.opbp;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.CommandLineRunner;
import org.springframework.stereotype.Component;

import br.com.eteg.opbp.entities.Account;
import br.com.eteg.opbp.repository.AccountRepository;

@Component
public class DatabaseLoader implements CommandLineRunner {

  @Autowired
  private AccountRepository userRepository;

  @Override
  public void run(String... args) throws Exception {
    userRepository.save(new Account("dougefr", "123456"));
  }
}
